ignite/cmd/chain.go

```diff
@@ -97,10 +102,67 @@ chain.
 	return c
 }
 
-func configMigrationPreRunHandler(cmd *cobra.Command, _ []string) (err error) {
+func preRunHandler(cmd *cobra.Command, _ []string) error {
 	session := cliui.New()
 	defer session.End()
 
+	if err := configMigrationPreRunHandler(cmd, session); err != nil {
+		return err
+	}
+	return toolsMigrationPreRunHandler(cmd, session)
+}
+
+func toolsMigrationPreRunHandler(cmd *cobra.Command, session *cliui.Session) (err error) {
+	session.StartSpinner("Checking missing tools...")
+
+	appPath := flagGetPath(cmd)
+	toolsFilename := filepath.Join(appPath, toolsFile)
+	f, _, err := xast.ParseFile(toolsFilename)
+	if err != nil {
+		return err
+	}
+
+	var (
+		missing = cosmosgen.MissingTools(f)
+		unused  = cosmosgen.UnusedTools(f)
+	)
+
+	session.StopSpinner()
+	if len(missing) > 0 {
+		question := fmt.Sprintf(
+			"Some imports are missing into the %s file: %s. Would you like to add?",
+			toolsFilename,
+			strings.Join(missing, ", "),
+		)
+		if err := session.AskConfirm(question); err != nil {
+			missing = []string{}
+		}
+	}
+
+	if len(unused) > 0 {
+		question := fmt.Sprintf(
+			"Some imports are unused into the %s file: %s. Would you like to remove?",
```

Comment:
If the imports are actually deprecated maybe:

```suggestion
			"File %s contains deprecated imports: %s. Would you like to remove them?",
```